python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -346,20 +391,13 @@ def upsert_athena_table(
             self.graph.emit(event)
         # jscpd:ignore-end
 
-        database_urn = "urn:li:container:" + "".join(
-            metadata.parent_entity_name.split()
-        )
+        database_urn = metadata.relationships[RelationshipType.PARENT][0].id
```

Comment:
This is assuming that the relationships list has at least one element - think there's an unhandled IndexError here